client/my-sites/checklist/checklist-header.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import ScreenReaderText from 'components/screen-reader-text';
+import ProgressBar from 'components/progress-bar';
+
+let globalIndex = 0;
+
+export class ChecklistHeader extends PureComponent {
+	static propTypes = {
+		total: PropTypes.number.isRequired,
+		completed: PropTypes.number.isRequired,
+		hideCompleted: PropTypes.bool,
+		onClick: PropTypes.func,
+	};
+
+	constructor( props ) {
+		super( props );
+		this.index = globalIndex++;
```

Comment:
Any way we can avoid the instance counter here? Do we expect to have many instances on the same page?

Reply:
No, we don't. I wanted to remove a potential error but it seems not necessary.